src/dbnode/storage/database.go

```diff
@@ -1067,6 +1083,99 @@ func (d *db) fetchIndexChecksum(
 	return n.FetchIndexChecksum(ctx, id, start)
 }
 
+func (d *db) ReadMismatches(
+	ctx context.Context,
+	namespace ident.ID,
+	query index.Query,
+	batchReader wide.IndexChecksumBlockBatchReader,
+	queryStart time.Time,
+	shards []uint32,
+	iterOpts index.IterationOptions,
+) ([]wide.ReadMismatch, error) { // TODO: update this type when reader hooked up
+	n, err := d.namespaceFor(namespace)
```

Comment:
Nit: rename to `ns` (`n` has too strong association with the number of elements of something; this triggers some confusion when seeing `n` passed to some method downstream, for example).

Reply:
`n` is pretty commonly namespace throughout this file, don't really want to get stuck renaming a lot of these in an unrelated PR, will change the var when its passed into the function though